markdown/articles/2021/12/24/mindkiller-update.md

```diff
@@ -0,0 +1,345 @@
+use this strategy if I trounce someone turn 1 and never see their deck.
+
+### 8Cast
+
+#### Out
+
+:::row{variant=hand}
+- Doomsday
+- Doomsday
+- Doomsday
+- Thassa's Oracle
+- Street Wraith
+:::
+
+#### In
+
+:::row{variant=hand}
+- Prismatic Ending
+- Prismatic Ending
+- Orim's Chant
+- Orim's Chant
+- Orim's Chant
+:::
+
+I'm not 100% sure of this sideboarding but I can't think of anything that might
+be better. There's tension here because Doomsday is generally good against
+prison decks but weaker against countermagic. Post-board, they tend to have more
+countermagic than prison elements, which pushed me away from Doomsday. While
+8Cast can sometimes put enough bodies on board to beat an Empty, I still feel
+like its a better plan than Doomsday. But due to the prison pieces, Galvanic
+Relay can be a liability, because they can drop a Chalice on the right number to
+turn off as much of your Relay gas as possible. So I ended up deciding that
+trading the Doomsday package for all the white disruption was the best plan.
+Since this means fewer business spells in the deck, I don't want to spend a Wish
+on a Prismatic Ending, so they're both in the deck.
+
+This matchup is the one where Abrupt Decays in the board might look good, but
+the reality is that they still won't shift the matchup to be winnable.
+
+## Thoughts and Prayers
+
+I'm pleased with how the deck has been performing lately. Being up on play
+points and tickets with a brew I've put a lot of time into, in a format as
+cutthroat as Legacy, definitely sparks joy. While the results aren't stunning, I
+feel confident taking the deck into leagues and would likewise feel confident
+taking it to local events. The high-variance nature makes me wary of bringing it
+to any event with more than 6 or so rounds, but people achieve strong result
+with Oops All Spells which seems similarly glass-cannon-ish.
+
+I also think this deck is substantially more fun than other combo decks of the
+same fast and loose style. Oops All Spells is known for its simplistic,
+repetitive gameplay. Belcher has more lines and decision points but ultimately
+boils down to firing off your hand and hoping for the best. I'm sure there's
+some bias in my beliefs as the deck's creator, but this deck gets the adrenaline
+pumping and the brain heated up unlike its peers. Spoils is a powerful card and
+exciting to play, and requires careful thought to balance risk and reward.
+Doomsday is strong and flexible, and using it to beat multiple on-board hate
+pieces is supremely fulfilling. And of course winning on turn 1 in style is the
+entire point of playing Magic. I'm sure this deck isn't for everyone but if
+you're an adrenaline junkie I encourage you once again to buy a playset of
+Spoils of the Vault. I hope I can convince you that this list is sufficiently
+powerful to compete in this format. Invoke the deity of your choice and spin the
+wheel.
```

Comment:
You missed the most important stat in the article, "Spoils Suicide Count"

Reply:
If only I had tracked that. I suspect it would be right in line with what the math says it should be though.